spec.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: page
+title: Semantic Specification
+---
+
+## Concepts and Terminology
+
+
+```
+Parentage/causal relationships for spans from a single trace
+
+        [Span A]  <--(the root span)
+            |
+     +------+------+
+     |             |
+ [Span B]      [Span C]
+     |             |
+ [Span D]      +---+-----+
+               |         |
+           [Span E]  [Span F]
+                         |
+                +--------+--------+
+                |        |        |
+            [Span G] [Span H] [Span I]
+
+```
+
+```
+Temporal relationships for spans from a single trace
+
+––|–––––––|–––––––|–––––––|–––––––|–––––––|–––––––|–––––––|–> time
+
+ [Span A···················································]
+   [Span B··············································]
+      [Span D··········································]
+    [Span C········································]
+         [Span E···········]   [Span F········]
+                                [Span G······]
+                                [Span H··]
+                                        [Span I·]
+```
+
+A **Trace** represents the potentially distributed, potentially concurrent data/execution path in a (potentially distributed, potentially concurrent) system. A Trace can be thought of as a tree of Spans. (See the ASCII diagrams above)
+
+A **Span** represents a logical unit of work in the system that has a start time and a duration. Spans may be nested and ordered to model parent-child and casual relationships. Each span has an **operation name**, a presumably human-readable string which concisely names the work done by the span (e.g., an RPC method name, a function name, or the name of a subtask within a larger computation).
+
+Every Span has zero or more **Logs**, each of which being a timestamped message with an optional structured data payload of arbitrary size.
+
+Every Span may also have zero or more key/value **Tags**, which do not have timestamps and simply annotate the spans.
+
```

Comment:
Maybe we can add: "Every Span has a unique identity determined by the *Trace Context*".